src/config/dynamic-theme-fixes.config

```diff
@@ -5198,7 +5198,7 @@ INVERT
 .gb_0b
 #dictionary-modules img[src*="png"]
 a.gb_b > div
-a[href*="about/products"]
```

Comment:
`a[href*="about/products"][title]` The google main page has a title the search one has aria-label. We should preserve this selector as going down to the normal class selector will be litterly cat and mouse.

Reply:
Previously done by name (for example, `a[title="Google apps"]`, but in different language versions of the search this name will be in a different language.

Reply:
That's why we don't add the ="" But simple the `[title]` That way it just check if it exist.